parse/parser.go

```diff
@@ -178,8 +178,18 @@ func Parse(buf TokenBuffer) (*ast.Program, error) {
 // TODO: implement me w/ test cases :-)
 func parseStatement(buf TokenBuffer) (ast.Statement, error) {
 	switch buf.Peek(CURRENT).Type {
+	case Assign:
+		return parseAssignStatement(buf)
+	case BoolType:
+		return parseAssignStatement(buf)
+	case If:
+		return parseIfStatement(buf)
+	case IntType:
+		return parseAssignStatement(buf)
 	case Return:
 		return parseReturnStatement(buf)
+	case StringType:
+		return parseAssignStatement(buf)
```

Comment:
Why don't we sort cases IntType, BoolType, StringType first. I mean sort Type cases first

Reply:
Oh, I sorted alphabetically.
I will modify it.